src/content_working_group/lib.rs

```diff
@@ -0,0 +1,2422 @@
+            hiring::DeactivateApplicationError::OpeningNotAcceptingApplications => {
+                "OpeningNotAcceptingApplications"
+            }
+            hiring::DeactivateApplicationError::UnstakingPeriodTooShort(_stake_purpose) => {
+                "UnstakingPeriodTooShort ..."
+            }
+            hiring::DeactivateApplicationError::RedundantUnstakingPeriodProvided(
+                _stake_purpose,
+            ) => "RedundantUnstakingPeriodProvided ...",
+        }
+    }
+}
+
+impl rstd::convert::From<WrappedError<members::ControllerAccountForMemberCheckFailed>> for &str {
+    fn from(wrapper: WrappedError<members::ControllerAccountForMemberCheckFailed>) -> Self {
+        match wrapper.error {
+            members::ControllerAccountForMemberCheckFailed::NotMember => "Is not a member",
+            members::ControllerAccountForMemberCheckFailed::NotControllerAccount => {
+                "Account is not controller account of member"
+            }
+        }
+    }
+}
+
+impl rstd::convert::From<WrappedError<hiring::AddApplicationError>> for &str {
+    fn from(wrapper: WrappedError<hiring::AddApplicationError>) -> Self {
+        match wrapper.error {
+            hiring::AddApplicationError::OpeningDoesNotExist => "OpeningDoesNotExist",
+            hiring::AddApplicationError::StakeProvidedWhenRedundant(_stake_purpose) => {
+                "StakeProvidedWhenRedundant ..."
+            }
+            hiring::AddApplicationError::StakeMissingWhenRequired(_stake_purpose) => {
+                "StakeMissingWhenRequired ..."
+            }
+            hiring::AddApplicationError::StakeAmountTooLow(_stake_purpose) => {
+                "StakeAmountTooLow ..."
+            }
+            hiring::AddApplicationError::OpeningNotInAcceptingApplicationsStage => {
+                "OpeningNotInAcceptingApplicationsStage"
+            }
+            hiring::AddApplicationError::NewApplicationWasCrowdedOut => {
+                "NewApplicationWasCrowdedOut"
+            }
+        }
+    }
+}
+
+impl rstd::convert::From<WrappedError<members::MemberControllerAccountDidNotSign>> for &str {
+    fn from(wrapper: WrappedError<members::MemberControllerAccountDidNotSign>) -> Self {
+        match wrapper.error {
+            members::MemberControllerAccountDidNotSign::UnsignedOrigin => "Unsigned origin",
+            members::MemberControllerAccountDidNotSign::MemberIdInvalid => "Member id is invalid",
+            members::MemberControllerAccountDidNotSign::SignerControllerAccountMismatch => {
+                "Signer does not match controller account"
+            }
+        }
+    }
+}
+
+// ======================================================================== //
+
+decl_storage! {
+    trait Store for Module<T: Trait> as ContentWorkingGroup {
+
+        /// The mint currently funding the rewards for this module.
+        pub Mint get(mint) config(): <T as minting::Trait>::MintId;
```

Comment:
we don't have configurable mints at genesis.. so setting a mintid at genesis is problematic atm.
Either we can have this be an `Option<MintId>` with the mint getting created later with a sudo call,
or my preferred way to use a build() with some extra_genesis values to create a new mint.

Reply:
It also just occurred to me, I don't see the mint id being used when creating reward relationships.

Reply:
I see now the relationship is created through an extrinsic where the mintid is provided..

Reply:
Here is one approach to create mint with build() at genesis: 438c031bfd429cc1f01e77ba8bf35c6c319dc138